manifests/cli-manager-operator.clusterserviceversion.yaml

```diff
@@ -101,34 +115,63 @@ spec:
           - list
           - create
           - update
+          - delete
+          - patch
         - apiGroups:
           - rbac.authorization.k8s.io
           resources:
           - clusterroles
           - clusterrolebindings
           verbs:
-          - '*'
+          - create
```

Comment:
The other clusterroles have `deletecollection` but this one has not. Is there any reason for this?

Reply:
it makes sense to add `deletecollection` here as well imo